app/src/components/group_schema_table.rs

```diff
@@ -154,7 +155,27 @@ impl GroupSchemaTable {
         let hardcoded = ctx.props().hardcoded;
         html! {
             <tr key={attribute.name.clone()}>
-                <td>{&attribute.name}</td>
+                <td>
+                    {&attribute.name}
+                    {
+                        match validate_attribute_name(&attribute.name) {
+                            Ok(()) => {
+                                html!{}
+                            }
+                            Err(_invalid_chars) => {
+                                html!{
+                                    <>
+                                        <br/>
+                                        <small class="text-warning">
+                                            {"Warning: This attribute uses one or more invalid characters. "}
+                                            {"Not globally supported."}
```

Comment:
Maybe "Some clients may not support it"?
Also, let's put the list of allowed characters there.

Reply:
Check. I'll see if I can find a good place for it.

Reply:
Done